crates/flo/src/types.rs

```diff
@@ -226,6 +226,11 @@ pub enum Statement {
     /// at snapshot time.
     Desnap(DesnapStatement),
 
+    /// An operation that takes a variable `source_var` of type `T1`, and
+    /// performs a semantic conversion of the data's type to be `T2`, outputting
+    /// the result in the `target_var`.
+    Transmute(TransmuteStatement),
```

Comment:
Can we use `ReinterpretBits`, or another name that means something to someone other than a rust programmer.?